bot/search/elastic_search_manager.py

```diff
@@ -117,13 +141,17 @@ async def main(logger: logging.Logger) -> None:
     es_client = await ElasticSearchManager.connect_to_elasticsearch(logger)
     try:
         await ElasticSearchManager.delete_all_indices(es_client, logger)
-        await ElasticSearchManager.index_transcriptions(base_path="../RANCZO-TRANSKRYPCJE", es=es_client, logger=logger)
+        base_path = Path("../RANCZO-TRANSKRYPCJE")
```

Comment:
a mówiłeś że nie masz `..` nigdzie xD

Reply:
w samym bocie nie XD to jest tylko do indeksowania do elastica przy podstawianiu foldera z transkrypcja dawno temu był ten main odpalany

Reply:
nwm jak to rozwiązać bo to jest dosłownie hardkodowana ścieżka to chyba jedynie żeby to było jako argument z terminala żeby odpalać to python asdf.py RANCZO-TRANSKYPCJE